db/schema.rb

```diff
@@ -48,17 +48,16 @@
     t.boolean "call_present"
     t.date "call_open_date"
     t.date "call_close_date"
-    t.string "intended_beneficiaries", array: true
```

Comment:
What makes these fields jump around, and could we separate this change from the others? I'm not saying do it or not, I'm trying to understand why sometimes we get in this cycle of different machines reordering existing fields in the schema. 😬

Reply:
Yeah, I don't know why it does that, it's annoying as hell. Usually when this happens, I just commit the relevant lines, but I was running on autopilot and forgot, so have edited the commit 👍